Signal/src/environment/PushRegistrationManager.swift

```diff
@@ -32,7 +32,7 @@ public class PushRegistrationManager: NSObject, PKPushRegistryDelegate {
     // Private callout queue that we can use to synchronously wait for our call to start
     // TODO: Rewrite call message routing to be able to synchronously report calls
     private static let calloutQueue = DispatchQueue(
-        label: "org.whispersystems.signal.PKPushRegistry",
```

Comment:
Everything below here was inherited from https://github.com/signalapp/Signal-iOS/pull/5423 and can be ignored for this PR.